frontend/src/components/VHeader/meta/VHeaderInternal.stories.ts

```diff
@@ -18,7 +18,7 @@ export const Default: Story = {
     setup() {
       return () =>
         h("div", { id: "teleports" }, [
-          h("div", { class: "fixed inset-0 w-full h-full" }, [
+          h("div", { class: "fixed inset-0 w-full h-full bg-default" }, [
```

Comment:
By default, there is no background, and the header background comes from the page background. Adding the background here to make the header look correct in both light and dark snapshots.